modules/byo-vpc-prep.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-prep_{context}"]
+= Preparing your BYO VPC
+
+.Procedure
+
+. Log in to your account on the link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html[AWS Management Console].
+. Create a VPC with the IP address of `10.0.0.0/16` and enable `dnsHostnames`.
+. Create an Internet gateway, and attach it to the VPC created in the last step.
```

Comment:
No detailed steps for the steps to create VPC

Reply:
@dofinn Do we have offer any suggestions on how to configure the VPC? I see this link/module that we could link to/reuse with some effort: https://docs.openshift.com/container-platform/4.6/installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.html#installation-custom-aws-vpc-requirements_installing-restricted-networks-aws-installer-provisioned

Reply:
The minimum requires are described in the SOP i shared with you regarding the :
* priv subnet
* nat
* pub subnet
* internet gateway
I guess its up to us as to how much detail we give customers in regards to setting this up in AWS.